External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -4063,8 +4068,15 @@ static public string GetCodeTipCode(ASResult result)
             {
                 ClassModel inClass = result.InClass ?? result.Type;
                 result.InFile = ASContext.Context.GetCodeModel(file);
-                inClass = result.InFile.GetClassByName(inClass.Name);
-                result.Member = inClass.Members.Search(result.Member.Name, 0, 0);
+                if (inClass.Name != "void")

```

Comment:
Thank you for the suggestion. I changed to comparing against `ClassModel.VoidClass`.